src/utils.rs

```diff
@@ -34,6 +35,28 @@ pub fn random_u32() -> u32 {
     OsRng.next_u32()
 }
 
+pub(crate) fn u32_to_bytes(d: &[u32]) -> Vec<u8> {
```

Comment:
I think we can reuse `codec.rs` for this:

https://github.com/raphaelrobert/maelstrom/blob/master/src/codec.rs#L119